polaris-react/src/components/Filters/components/SearchField/SearchField.tsx

```diff
@@ -35,19 +33,14 @@ export function SearchField({
   placeholder,
   disabled,
   borderlessQueryField,
+  showQueryIcon,
   loading,
 }: SearchFieldProps) {
-  const i18n = useI18n();
   const id = useId();
-  const inputRef = useRef<HTMLInputElement>(null);
   function handleChange(value: string) {
     onChange(value);
   }
 
-  useEffect(() => {
-    if (focused) inputRef.current?.focus();
-  }, [focused]);
-
```

Comment:
this logic also lives in [`TextField`](https://github.com/Shopify/polaris/blob/main/polaris-react/src/components/TextField/TextField.tsx#L268-L272), I'm passing `focused` onto `TextField` below